orttraining/orttraining/python/training/debug.py

```diff
@@ -0,0 +1,33 @@
+
+import numpy as np
+import os
+import sys
+import torch
+
+from numpy.testing import assert_allclose
+
+# Inputs are the state_dicts for each of the model (dictionary for each layer and weights)
+def compare_onnx_weights(model_a, model_b, verbose=False, rtol=1e-4):
```

Comment:
Write this method so it takes two ORTTrainer instances directly rather than expecting the user to pass trainer._training_session.get_state()